### DISCUSSION TITLE
Phishing Campaign active via vercel.app

### DISCUSSION THREAD

**Discussion by jonathanarbely** [vercel/community#6273] [Help] [answered]
### Summary

FYI: There's an active fraud campaign running live on Vercel, please remove. The site is trying to steal IONOS login data.

**Comment by amyegan** [comment, marked as answer]
Hey @jonathanarbely. Thanks for reporting this! If you haven't already, please send a copy of this report to [abuse@vercel.com](mailto:abuse@vercel.com) so it can be properly tracked in our system. I'll let the team know to be on the lookout for your email 🙇

**Reply by jonathanarbely** [reply]
Done, thanks.

**Reply by amyegan** [reply]
The team let me know they received your report. Thanks again for letting us know!
